### PR TITLE
First draft documentation readme

### PR DESCRIPTION
**Pull request by nshiab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,113 @@
...
+
+## Core principles
+
+The library expects **tabular data** stored in CSV files or **arrays of objects** stored in JSON files. It works best when the data is tidy:
+
+1. Every column (or key) is a variable
+
+2. Every row (or item) is an observation
+
+3. Every cell (or value) is a single value
```
Comment (isabellebouchard) [inline README.md:28]: Perhaps we should give an example? 

```suggestion

1. Every column (or key) is a variable 

2. Every row (or item) is an observation

3. Every cell (or value) is a single value

For instance, for hourly meteorological data, our tidy dataset would look like this.  
time         day            temperature   precipitation
00:00:00     2022-05-02      15.2          0
01:00:00     2022-05-02      15.9.         0
02:00:00     2022-05-02      16.2.         0
03:00:00     2022-05-02      18.0          0.5
04:00:00     2022-05-02      19.3.         0
05:00:00     2022-05-02      19.4.         0
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+```
+
+But you can also import everything if you wish.
+```js
+import * as sda from "simple-data-analysis"
+
+const simpleData = sda.createSimpleData(someData)
+```
+
+## SimpleDocument (experimental, for NodeJS only)
```
Comment (isabellebouchard) [inline README.md:84]: I would have a similar section for `SimpleData`. You could show how you can easily chain! 🎉

**Review by isabellebouchard — APPROVED** [review]
Nice first draft. 

I think that we should point to our examples folder in the README. 

Also, there is a `SimpleDocument` example, but none of `SimpleData`. Would be great to have both.

File: README.md (hunk 1 above)
Comment (nshiab) [reply README.md:28]: I think I'll link to a resource explaining it. Otherwise, it could be long before we actually get to the code!

**Comment by nshiab** [comment]
I've adjusted everything. I am pushing to main to see if the readme renders well on github. (we will recheck everything together before v1.0.0)

**Pull request merged**